DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfigurationController.cs

```diff
@@ -116,5 +119,100 @@ public IActionResult EditCentreWebsiteDetails(EditCentreWebsiteDetailsViewModel
 
             return RedirectToAction("Index");
         }
+
+        [HttpGet]
+        [Route("RegistrationPrompts/{promptNumber}/Edit")]
+        public IActionResult EditRegistrationPrompt(int promptNumber)
+        {
+            var centreId = User.GetCentreId();
+
+            var customPrompt = customPromptsService.GetCustomPromptsForCentreByCentreId(centreId).CustomPrompts
+                .Single(cp => cp.CustomPromptNumber == promptNumber);
+
+            return View(new EditRegistrationPromptViewModel(customPrompt));
+        }
+
+        [HttpPost]
+        [Route("RegistrationPrompts/{promptNumber}/Edit")]
+        public IActionResult EditRegistrationPrompt(EditRegistrationPromptViewModel model, string action)
+        {
+            if (action.StartsWith("delete") && int.TryParse(action.Remove(0, 6), out var index))
+            {
+                return EditRegistrationPromptPostRemovePrompt(model, index);
+            }
+
+            return action switch
+            {
+                "save" => EditRegistrationPromptPostSave(model),
+                "addPrompt" => EditRegistrationPromptPostAddPrompt(model),
+                _ => View(model)

```

Comment:
Should we redirect to an error page (see LearningSolutions controller) if we can't find a match for the action?